src/system/Table/TableCell.js

```diff
@@ -0,0 +1,57 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { Heading, Box } from '../';
+
+const TableCell = ( { head, isRowHead, children, ...rest } ) => {
+	let scope = null;
+
+	if ( head ) {
```

Comment:
@brunobasto unrelated to this PR but I find the "head/isRowHead" a little bit confusing. Even by looking at the code I always wonder what they are used for.

I'm sure renaming them might not be the case now but I was wondering if we could add some extra example in the stories to clarify the use of the parameters

Reply:
I agree it's confusing. I'll play with the storybook and add some examples.

Reply:
So, essentially, `head` and `isRowHead` control two things:

1. Whether to render the columns as `<td>` or `<th>`. If either is true, the column will render as `<th>`
2. The scope of the column header:

> The scope attribute specifies whether a header cell is a header for a column, row, or group of columns or rows.
> 
> **Note**: The scope attribute has no visual effect in ordinary web browsers, but can be used by screen readers. 
> 
> https://www.w3schools.com/tags/att_th_scope.asp

I searched in `vip-ui` and couldn't find any usages of `rowHead` or `isRowHead`, so, for the sake of simplicity in the API, I [removed](https://github.com/Automattic/vip-design-system/pull/74/commits/e5e87840180bc9aa4f23e65609dd963d0a117fe3) `isRowHead` and, since we pass forward the props, if someone ever needs to control the scope for accessibility purposes, they can do:

```jsx
<TableCell head scope="row">...
```

Reply:
Since the prop I removed was unused (so it shouldn't impact the Dashboard) and the PR is already approved, I'll be bold and move forward with merging this, but we can add it back quickly if we need to.